### PR TITLE
query multiple tables

### PR DESCRIPTION
**Pull request by jheart-vic**
- Created a table named `owners` with the following columns:
  - `id`: integer
  - `full_name`: string
  - `age`: integer
- Created a table named `species` with the following columns:
  - `id`: integer 
  - `name`: string
- Modified `animals` table:
  - Made sure that id is set as autoincremented PRIMARY KEY
  - Removed column `species`
  - Added column `species_id` which is a foreign key referencing `species` table
  - Added column `owner_id` which is a foreign key referencing the `owners` table
- Inserted the following data into the `owners` table:
  - Sam Smith 34 years old.
  - Jennifer Orwell 19 years old.
  - Bob 45 years old.
  - Melody Pond 77 years old.
  - Dean Winchester 14 years old.
  - Jodie Whittaker 38 years old.
 - Inserted the following data into the `species` table:
   - Pokemon
   - Digimon
   - Modify your inserted animals so it includes the `species_id` value:
  - If the name ends in "mon" it will be Digimon
  - All other animals are Pokemon
- Modified the inserted animals to include owner information (`owner_id`):
  - Sam Smith owns Agumon.
  - Jennifer Orwell owns Gabumon and Pikachu.
  - Bob owns Devimon and Plantmon.
  - Melody Pond owns Charmander, Squirtle, and Blossom.
  - Dean Winchester owns Angemon and Boarmon.
 - Here is the screenshot
Screenshot (68)

- Wrote queries (using `JOIN`) to answer the following questions:
  - What animals belong to Melody Pond?
  - List of all animals that are pokemon (their type is Pokemon).
  - List all owners and their animals, remember to include those that don't own any animal.
  - How many animals are there per species?
  - List all Digimon owned by Jennifer Orwell.
  - List all animals owned by Dean Winchester that haven't tried to escape.
  - Who owns the most animals? 
 
Screenshot (71)

Screenshot (72)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+CREATE TABLE owners (
+  id serial PRIMARY KEY,
+  full_name varchar(100),
+  age int
+);
+
+CREATE TABLE species (
+  id serial PRIMARY KEY,
+  full_name varchar(100),
+  age int
```
Comment (Munsa1) [inline schema.sql]: - [ ] The species table has no `Age` column, kindly remove it.

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+);
+
+ALTER TABLE animals
+  ADD species_id int
```
Comment (Munsa1) [inline schema.sql]: - [ ] The `species_id` references the species table kindly add the referential integrity constraint.
e,g
``ALTER TABLE animals ADD species_id int REFERENCES species(id);``
- [ ] Its also a requirement that you add the column `owner_id` which is a foreign key referencing the owners table, kindly do so.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @jheart-vic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @jheart-vic 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**